sdk/servicebus/azure-servicebus/tests/servicebus_preparer.py

```diff
@@ -47,6 +47,8 @@ def __init__(self,
 
         self.set_cache(use_cache, sku, location)
 
+        self.set_cache(use_cache, sku, location)
```

Comment:
This got duplicated in the merge.  It's idempotent, so not breaking anything, but should get yanked.